src/utils/context.js

```diff
@@ -42,7 +42,12 @@ export class Context {
     getColorValue(color) {
         const matchedColor = this.container.findColorEqual(color, this.useLinkedStyleguides);
         if (matchedColor) {
-            return `colors.${matchedColor.getFormattedName("constant")}`;
+            const colorName = (
+                matchedColor.getFormattedName
+                    ? matchedColor.getFormattedName("constant")
```

Comment:
The comment I put in stylesheet-extensions repo applies to this one as well:
https://github.com/zeplin/stylesheet-extensions/pull/38#discussion_r592648754